integration-cli/docker_experimental_network_test.go

```diff
@@ -0,0 +1,589 @@
+// +build experimental
+
+package main
+
+import (
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/pkg/integration/checker"
+	"github.com/docker/docker/pkg/parsers/kernel"
+	"github.com/go-check/check"
+)
+
+var (
+	MacvlanKernelSupport = testRequirement{
```

Comment:
Our minimum supported kernel is 3.10, so not sure we need this test requirement.